### PR TITLE
Feature/473 msb example

### PR DESCRIPTION
**Pull request by sastafford**
Provide example of bringing in data from RDBMS in H2 into Data Hub.

@paxtonhare - the ingest and harmonize flows are not doing anything in this example, would like to discuss with you once this gets merged in.

### CODE REVIEW

File: examples/spring-batch/src/test/resources/job.properties [hunk 1]
```diff
@@ -0,0 +1,13 @@
+marklogic.host=oscar
```
Comment (paxtonhare) [inline examples/spring-batch/src/test/resources/job.properties]: can you make these hosts localhost? it's likely the default for most users.

File: examples/spring-batch/README.md [hunk 2]
```diff
@@ -4,26 +4,23 @@ This example demonstrates how to run a custom Spring Batch job against the Data
 
 Learning [Spring Batch](http://docs.spring.io/spring-batch/reference/html/spring-batch-intro.html) is beyond the scope of this README. But let's pretend you know enough to be dangerous.
 
-Now you want to use Spring Batch to load a bunch of data into MarkLogic. Maybe that data is coming from a message queue. Maybe it's a bunch of files in a folder. It can be anything really. 
+Now you want to use Spring Batch to load a bunch from a relational database into MarkLogic.
```
Comment (paxtonhare) [inline examples/spring-batch/README.md]: make this "to load data from a relational database into MarkLogic."

**Comment by paxtonhare** [comment]
Can you put that diagram picture in the readme too?

**Comment by sastafford** [comment]
@paxtonhare - changes made to README and properties files

**Pull request merged**